commit 57878380fbe05fb4c2079520da8dfd0f8ddf9021
Author: Yingzhong.Zha <yzha@codeaurora.org>
Date:   Mon Mar 20 15:00:36 2017 +0800

    [PATCH] Update header file with 4.4 kernel header.
    
    This is to resolve glibc-2.23 compilation issue
    with 4.4 kernel.
    
    Change-Id: I32a62a25cd30a78efbf9c3a32744ab7368e63e6d
    Signed-off-by: Yingzhong.Zha <yzha@codeaurora.org>

diff --git a/arch/arm64/include/uapi/asm/sigcontext.h b/arch/arm64/include/uapi/asm/sigcontext.h
index c731ca0..ee469be 100644
--- a/arch/arm64/include/uapi/asm/sigcontext.h
+++ b/arch/arm64/include/uapi/asm/sigcontext.h
@@ -16,7 +16,6 @@
 #ifndef _UAPI__ASM_SIGCONTEXT_H
 #define _UAPI__ASM_SIGCONTEXT_H
 
-#ifdef CONFIG_64BIT
 #include <linux/types.h>
 
 /*
@@ -62,35 +61,4 @@ struct esr_context {
 	__u64 esr;
 };
 
-#else /* CONFIG_64BIT */
-
-/*
- * Signal context structure - contains all info to do with the state
- * before the signal handler was invoked.  Note: only add new entries
- * to the end of the structure.
- */
-struct sigcontext {
-	unsigned long trap_no;
-	unsigned long error_code;
-	unsigned long oldmask;
-	unsigned long arm_r0;
-	unsigned long arm_r1;
-	unsigned long arm_r2;
-	unsigned long arm_r3;
-	unsigned long arm_r4;
-	unsigned long arm_r5;
-	unsigned long arm_r6;
-	unsigned long arm_r7;
-	unsigned long arm_r8;
-	unsigned long arm_r9;
-	unsigned long arm_r10;
-	unsigned long arm_fp;
-	unsigned long arm_ip;
-	unsigned long arm_sp;
-	unsigned long arm_lr;
-	unsigned long arm_pc;
-	unsigned long arm_cpsr;
-	unsigned long fault_address;
-};
-#endif /* CONFIG_64BIT */
 #endif /* _UAPI__ASM_SIGCONTEXT_H */
